asistente_ladm_col/logic/quality_rules/qr_building_duplicate_records.py

```diff
@@ -0,0 +1,97 @@
+"""
+/***************************************************************************
+                              Asistente LADM_COL
+                             --------------------
+        begin                : 2022-06-15
+        git sha              : :%H$
+        copyright            : (C) 2022 by Sergio Ramírez (BSF Swissphoto)
+        email                : seralra96@gmail.com
+ ***************************************************************************/
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License v3.0 as          *
+ *   published by the Free Software Foundation.                            *
+ *                                                                         *
+ ***************************************************************************/
+"""
+from qgis.PyQt.QtCore import (QCoreApplication,
+                              QSettings)
+
+from asistente_ladm_col.config.enums import EnumQualityRuleResult
+from asistente_ladm_col.config.keys.common import QUALITY_RULE_LADM_COL_LAYERS
+from asistente_ladm_col.config.layer_config import LADMNames, LayerConfig
+from asistente_ladm_col.config.quality_rule_config import (QR_IGACR4016,
+                                                           QRE_IGACR4016E01)
+from asistente_ladm_col.core.quality_rules.abstract_logic_quality_rule import AbstractLogicQualityRule
+from asistente_ladm_col.core.quality_rules.quality_rule_execution_result import QualityRuleExecutionResult
+from asistente_ladm_col.logic.ladm_col.ladm_data import LADMData
+
+
+class QRDuplicateBuildingRecords(AbstractLogicQualityRule):
+    """
+    Check that building don't have duplicate records
+    """
+    _ERROR_01 = QRE_IGACR4016E01  # Building with duplicate records
+
+    def __init__(self):
+        AbstractLogicQualityRule.__init__(self)
+
+        self._id = QR_IGACR4016
+        self._name = "Revisar que la tabla construccion no tenga registros repetidos"
+        self._tags = ["igac", "instituto geográfico agustín codazzi", "lógica", "negocio", "construccion", "registros repetidos"]
+        self._models = [LADMNames.SURVEY_MODEL_KEY]
+
+        self._errors = {self._ERROR_01: "Construccion no debe tener registros repetidos"}
```

Comment:
add tilde "construcción"

Reply:
fixed in https://github.com/SwissTierrasColombia/Asistente-LADM-COL/pull/547/commits/955b7fa942b033b1d5edf8d4f3d6ce7a590ca9da